docs/examples/accounts

```diff
@@ -41,6 +41,7 @@
 #    ;stunpass=STUN/TURN/ICE-password
 #    ;stunserver=stun:[user:pass]@host[:port]  # see RFC 3986: Use of the format "user:password" in the userinfo field is deprecated.
 #    ;tcpsrcport=6060
+#    ;uas_req={yes, no, tls}  # default: no
```

Comment:
the word `UAS` is coming from RFC 3261 and is a technical term that should not be
exposed to the enduser.

Perhaps a better term is "incoming request" or "inbound request" ...?

Reply:
Works for me. Would you prefer it if I renamed it for all internal cases as well? E.g. `account_set_incoming_req_mode` instead of `account_set_uas_req_mode`, `UAS_REQ_MODE_OFF` ->`INCOMING_REQ_MODE_OFF`.

Reply:
Should we also change the current `uas_user` and `uas_pass` ?
```
uas_req  --> inreq_allowed = {yes, no}
uas_user --> inreq_user
uas_pass --> inreq_pass
```
We already removed `inreq_allowed=tls` and will replace this use-case with `sip_transports=tls`.